app/views/coach/edit.scala

```diff
@@ -1,19 +1,18 @@
 package views.html.coach
 
+import controllers.routes
 import play.api.data.Form
 import play.api.libs.json.Json
 
-import lila.app.templating.Environment.{ given, * }
-import lila.i18n.LangList
+import lila.app.templating.Environment.{ *, given }
 import lila.app.ui.ScalatagsTemplate.{ *, given }
-import lila.common.String.html.{ richText, safeJsonValue }
-
-import controllers.routes
+import lila.common.String.html.safeJsonValue
+import lila.i18n.LangList
 
 object edit:
 
-  private val dataTab   = attr("data-tab")
-  private val dataValue = attr("data-value")
+  private val dataTab = attr("data-tab")
+  attr("data-value")
```

Comment:
this is weird

Reply:
It could've deleted the whole line, but decided to just remove the assigning part 🤷.